### ISSUE TITLE
No cookie

### ISSUE THREAD

**Issue by imperugo** [vesse/passport-ldapauth#42]
Hi, 
I've updated my code to `passport 0.3.2` (previously I was on 0.2.1) and `passport-ldapauth 0.5.0` (previously I was on 0.2.4) and now, after the login, I don't have the session cookie.
If I revert to the previous version it works.

Any hit?

Thx

**Comment by vesse** [comment]
Nope, I'm using both versions successfully. Cookie is not even set by either of those libraries. Often `express-session` is used and [it sets the cookie](https://github.com/expressjs/session/blob/005078aaada837d2a57e7324986ead60bf326e84/index.js#L587).
